primerj/src/main/java/org/primer/primerj/core/TxBuilder.java

```diff
@@ -515,30 +516,21 @@ public Tx buildTx(String changeAddress, List<Out> unspendOuts, Tx tx, Coin... co
         for (Out out : tx.getOuts()) {
             value += out.getOutValue();
         }
-        boolean needMinFee = TxBuilder.needMinFee(tx.getOuts());
 
         if (value != TxBuilder.getAmount(unspendOuts) || value != TxBuilder.getAmount(outs)) {
             return null;
         }
 
         long fees = 0;
-        if (needMinFee) {
-            if (coin != null && coin.length > 0) {
-                fees = coin[0].getSplitNormalFee();
-            } else {
-                fees = Utils.getFeeBase();
-            }
+        if (coin != null && coin.length > 0) {
+            fees = coin[0].getSplitNormalFee();
         } else {
-            // no fee logic
-            int s = TxBuilder.estimationTxSize(outs.size(), tx.getOuts().size());
-            if (TxBuilder.getCoinDepth(outs) <= TxBuilder.TX_FREE_MIN_PRIORITY * s) {
-                fees = Utils.getFeeBase();
-            }
+            fees = Utils.getFeeBase();
```

Comment:
similarly, no use for us, also you are setting the fees immediately below so this is redundant